src/hotspot/cpu/aarch64/aarch64_vector.ad

```diff
@@ -2854,11 +2854,11 @@ instruct reduce_addL_sve(iRegLNoSp dst, iRegL isrc, vReg vsrc, vRegD tmp) %{
 %}
 
 // reduction addF
```

Comment:
Well, under "Vector reduction add" title, we have sub titles for reductions of various types. For ex. "reduction addI" for integers, "reduction addL" for Longs etc. It might be good to follow this format and also keep "reduction addF" for floats and "reduction addD" for Doubles. What do you think?

Reply:
I would vote to eventually remove all of them, they really don't add value. But I don't care too much, so feel free to leave them.